script/corpusfile2trec.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+import os
+from argparse import ArgumentParser
+
+TREC_FORMAT = "<DOC>\n<DOCNO>{}</DOCNO>\n<TEXT>\n{}</TEXT>\n</DOC>\n"
+
+def compute_paths(path):
+    paths = []
+    for root, _, fnames in os.walk(path):
+        paths.extend([os.path.join(root, fname) for fname in fnames])
+    return paths
+
+def to_trec(paths, output):
+    with open(output, "w") as f:
+        for path in paths:
+            f.write(TREC_FORMAT.format(path, open(path).read()))
+
+def main():
+    parser = ArgumentParser(description="Formats a corpus file as TREC.")
+    parser.add_argument("input", type=str, help="Input directory")
+    parser.add_argument("output", type=str, help="Output filename")
+    args = parser.parse_args()
+    print("Formatting '{}' to TREC...".format(str(args.input)))
+    to_trec(compute_paths(args.input), args.output)
+
+if __name__ == "__main__":
+    main()
```

Comment:
What are the assumptions here? One file corresponds to a new TREC document?
Should we call this script `files2trec` or something like that?

Reply:
Yes, one file corresponds to a new TREC doc (@elshize). I think that files2trec is a shortest name, but maybe it can be generic ¿docfiles2trec?

Reply:
I prefer `files2trec` rather than `docfiles2trec`. 

Also, I would add in the documentation that:
- one file corresponds to a new TREC document
- the filename corresponds to the document ID

Reply:
@gustingonzalez can you rename this?